docs/src/content/docs/id/guides/customization.mdx

```diff
@@ -123,10 +132,10 @@ Anda dapat menyesuaikan — atau bahkan menonaktifkan — daftar isi secara glob
 
 Secara default, heading `<h2>` dan `<h3>` akan dimasukkan ke dalam daftar isi. Ubah tingkat heading yang akan dimasukkan di seluruh website menggunakan opsi `minHeadingLevel` dan `maxHeadingLevel` dalam konfigurasi [global `tableOfContents`](/id/reference/configuration/#tableofcontents). Timpa pengaturan default ini pada masing-masing halaman dengan menambahkan properti [frontmatter `tableOfContents`](/id/reference/frontmatter/#tableofcontents) yang sesuai.
 
-<Tabs>
+<Tabs syncKey="config-type">
```

Comment:
Even tho this change is correct, the tabs are not synced as it requires identical `label`s. For the first set of tab, the second tab item label is `Konfigurasi global` and for the second one, it's `Konfigurasi Global` (notice the `g` / `G` difference). To get synced tabs, they both should be identical.

I'll let you pick the one that makes the most sense to you.

Reply:
I see, will adjust so it will sync. Thank you.